framework/rbac/BaseManager.php

```diff
@@ -220,4 +220,15 @@ protected function executeRule($user, $item, $params)
             throw new InvalidConfigException("Rule not found: {$item->ruleName}");
         }
     }
+
+    /**
+     * Check that there no any roles in user's role assignments and in default roles
```

Comment:
Method checks, whether $assignments array is empty and [[defaultRoles]] are empty as well.